forest/src/cli/config.rs

```diff
@@ -12,6 +12,9 @@ pub struct Config {
     pub data_dir: String,
     pub genesis_file: Option<String>,
     pub drand_dist_public: DistPublic,
+    pub enable_rpc: bool,
+    pub rpc_port: Option<String>,
+    pub rpc_listen: Option<String>,
```

Comment:
These don't need to be wrapped in an option, the default implementation is sufficient (this can never be None, the only reason the genesis file can be is when it uses the builtin genesis)